mythril/laser/ethereum/instruction_data.py

```diff
@@ -210,7 +212,9 @@ def calculate_native_gas(size: int, contract: str):
     elif contract == "identity":
         gas_value = opcodes.GIDENTITYBASE + word_num * opcodes.GIDENTITYWORD
     else:
-        raise ValueError("Unknown contract type {}".format(contract))
+        # TODO: Add gas for other precompiles, computation should be shifted to natives.py
+        #  as some data isn't available here
+        pass
```

Comment:
Now gas_value will contain a None value. Could this break callee's that expect an integer?

Reply:
Yep, But this was actually fixed in the develop, should merge with it.